.github/actions/stale_issues/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Filter stale issues
+inputs:
+  token:
+    description: 'The GITHUB_TOKEN secret'
+    required: true
+  period:
+    description: "The stale period in days"
+    required: false
+    type: number
+    default: 14
```

Comment:
I'd remove this default. Better to have it set explicitly. And we're not even using it in `stale-issues.yml`.